pages/success/index.js

```diff
@@ -107,6 +107,12 @@ const getAdditionalInfo = ({ task, project }) => {
   }
 };
 
+const getProjectLink = ({ project }) => {
+  const projectId = get(project, 'id');
+  const establishmentId = get(project, 'establishment.id');
+  return `establishments/${establishmentId}/projects/${projectId}`;
```

Comment:
It's probably better to do `req.buildRoute('project.read', { projectId, establishmentId })` here so that it's automatically consistent with the routing configuration.

Reply:
Have updated it to be more consistent